### PR TITLE
Matched func_800968B1

### PR DESCRIPTION
**Pull request by MaxBartlett**
Matched func_800968B1

### CODE REVIEW

File: src/boot_O2_g3/boot_0x800968B0.c [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
 u32 func_800968B0(const u8* a0, const u8* a1) {
     u8 v0;
     u8 v1;
     do {
         v0 = *a0++;
         v1 = *a1++;
         if (v0 != v1) {
-			// TODO this uses v0 instead of a3
-            return v0 - v1;
+	  return v0 - v1;
```
[the remark this answers is not in the document]
Comment (MaxBartlett) [reply src/boot_O2_g3/boot_0x800968B0.c:14]: Looks like it's still improperly spaced... let me try again.

**Comment by Kenix3** [comment]
ROM compiles OK. Approved.
